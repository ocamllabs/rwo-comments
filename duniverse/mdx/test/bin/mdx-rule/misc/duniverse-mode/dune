(rule
 (target dune.gen)
 (action
  (with-stdout-to
   %{target}
   (run ocaml-mdx rule --duniverse-mode --prelude %{dep:prelude.ml}
     --prelude-str "#require \"prelude-str\"" %{dep:duniverse-mode.md}))))

(rule
 (alias runtest)
 (action
  (diff %{dep:dune.gen.expected} %{dep:dune.gen})))
