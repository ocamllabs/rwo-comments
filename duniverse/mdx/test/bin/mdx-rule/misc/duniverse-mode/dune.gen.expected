(alias
 (name runtest)
 (deps
  (:x duniverse-mode.md)
  (package block)
  (package mdx)
  (package prelude-ml)
  (package prelude-str)
  prelude.ml)
 (action
  (progn
   (run ocaml-mdx test --prelude=prelude.ml --prelude-str
     "#require \"prelude-str\"" %{x})
   (diff? %{x} %{x}.corrected))))
